components/ActionSheet/src/MDCActionSheetController.h

```diff
@@ -203,6 +203,21 @@ __attribute__((objc_subclassing_restricted)) @interface MDCActionSheetController
  */
 @property(nonatomic) UIImageRenderingMode imageRenderingMode;
 
+/**
+ Determines if a divider should be shown between the header and actions. To customize the divider
+ color see @c headerDividerColor.
+
+ Defaults to NO.
+ */
+@property(nonatomic, assign) BOOL showHeaderDivider;
+
+/**
+ The color of the divider between the header and actions.
+
+ Defaults to UIColor.clearColor.
+ */
+@property(nonatomic, copy, nonnull) UIColor *headerDividerColor;
```

Comment:
I don't really understand why there are two separate properties controlling the visibility for this view. Could you help me understand why the two properties are necessary?

Reply:
This is following the convention set in #7567, I think this is what @wenyuzhang666 and @jverkoey would prefer.

Reply:
While I don't believe the boolean is strictly necessary, in #7567 the default value is visible. It's documented as "light grey". In this case it appears the default color is clear.  In order to use the property on Banner the user only needs to toggle the boolean flag. However here, they need to toggle the boolean *and* set a non-clear color. I think there needs to be a visible default color for the divider so users don't have to take two steps to enable the divider.

I'd also like to note that in [the relevant conversation thread for #7567](https://github.com/material-components/material-components-ios/pull/7567/files#r293004510), having both properties is referred to as an "acceptable path forward". This would seem like there was a compromise made at some point and the final decision was a result of that compromise.  Unfortunately the reasons for the decision aren't entirely clear, except a reference to theming extensions. I would be cautious about using that decision as the only basis for this one.

Reply:
I've added a default color to match Banner.